eternalcore/src/main/java/com/eternalcode/core/configuration/language/LanguageConfiguration.java

```diff
@@ -16,8 +22,74 @@ public class LanguageConfiguration implements ReloadableConfig {
 
     public LanguageSelector languageSelector = new LanguageSelector();
 
+    @Contextual
+    public static class LanguageSelector  {
+
+        @Description({ StringUtils.EMPTY, "# Name of inventory" })
+        public String title = "&6Select a language";
+
+        @Description({ StringUtils.EMPTY, "# Size of inventory" })
+        public int rows = 5;
+
+        @Description({ StringUtils.EMPTY, "# Border settings" })
+        public Border border = new Border();
```

Comment:
Przy okazji jak tu jesteś to zmień wszystkie StringUtils.EMPTY na " "

Przykład 👇
```suggestion
        @Description({ " ", "# Name of inventory" })
        public String title = "&6Select a language";

        @Description({ " ", "# Size of inventory" })
        public int rows = 5;

        @Description({ " ", "# Border settings" })
        public Border border = new Border();
```